docs/analytics/export.md

```diff
@@ -183,15 +183,33 @@ The table below shows the field mapping for the "customDimensions" field.
 
 | Application Insights            | App Center                                     |
 | ------------------------------- | ---------------------------------------------- |
-|  MessageType                    | Type of event (session, event, ...)            |
+|  AppBuild                       | Application build number                       |
 |  AppId                          | App Center App ID                              |
-|  CarrierCountry                 | Carrier Country                                |
-|  TimeZoneOffset                 | Time Zone Offset                               |
-|  CarrierName                    | Carrier Type                                   |
+|  AppNamespace                   | Application namespace                          |
+|  CarrierCountry                 | Carrier country                                |
```

Comment:
whats the difference between carrierCountry and countryCode? SDK sends only carrierCountry.

Reply:
Must be internal mapping. Removing CountryCode.